Packs/McAfee-TIE/Integrations/McAfee-TIE_V2/McAfee-TIE_V2.py

```diff
@@ -0,0 +1,603 @@
+        'Provider ID': provider_id,
+        'Provider (verbose)': provider,
+        'Created date': create_date_str
+    }
+
+    return res
+
+
+def parse_reference_human_readable(reference):
+    agent_guid = reference.get(FirstRefProp.SYSTEM_GUID)
+    date = reference.get(FirstRefProp.DATE)
+
+    return {
+        'Date': epoch_to_localtime(date),
+        'AgentGuid': agent_guid.replace('{', '').replace('}', '')  # Remove brackets if exist
+    }
+
+
+def reputations_to_human_readable(reputations):
+    return [parse_reputation_human_readable(rep) for rep in reputations]
+
+
+def references_to_human_readable(references):
+    return [parse_reference_human_readable(ref) for ref in references]
+
+
+def trust_level_to_score(trust_level):
+    if (trust_level >= TrustLevel.MIGHT_BE_TRUSTED):
+        return Common.DBotScore.GOOD
+
+    elif (trust_level == TrustLevel.MIGHT_BE_MALICIOUS):
+        return Common.DBotScore.SUSPICIOUS
+
+    elif (trust_level == TrustLevel.NOT_SET or trust_level == TrustLevel.UNKNOWN):
+        return Common.DBotScore.NONE
+
+    elif (trust_level < TrustLevel.MIGHT_BE_MALICIOUS):
+        return Common.DBotScore.BAD
+
+    else:
+        # Shouldn't reach here, as the API doesn't support 31-69 values except for 50)
+        return Common.DBotScore.NONE
+
+
+def test_module(dxl_client: DxlClient):
+    """Tests if there is a connection with DxlClient(which is used for connection with McAfee TIE, instead of the Client class)"""
+    dxl_client.connect()
+    return 'ok'
+
+
+def safe_get_file_reputation(tie_client: TieClient, api_input: Dict[str, str]):
+    try:
+        res = tie_client.get_file_reputation(api_input)
+    except Exception as e:
+        demisto.info(f'McAfee failed to get file reputation with error: {str(e)}')
+        return None
+    return res
+
+
+def get_hash_type_key(file_hash: str):
+    hash_type = get_hash_type(file_hash)
+    hash_type_key = HASH_TYPE_KEYS.get(hash_type, None)
+    if not hash_type_key:
+        raise DemistoException(f'Invalid value, {file_hash} is not a valid SHA1, SHA256 or MD5 value.')
+    return hash_type_key
+
+
+def get_file_instance(dbot_score: Union[Common.DBotScore, None], file_hash: str, hash_type_key: str) -> Common.File:
+    return Common.File(dbot_score=dbot_score,
+                       md5=file_hash if hash_type_key == HashType.MD5 else None,
+                       sha1=file_hash if hash_type_key == HashType.SHA1 else None,
+                       sha256=file_hash if hash_type_key == HashType.SHA256 else None)
+
+
+def parse_reputation(provider_id: int, reputation: Dict):
+    file_reputation_parser = GeneralFileReputationParser.init(provider=provider_id)
+    parsed_reputation = file_reputation_parser.parse_data(reputation_data=reputation)
+    parsed_reputation['Provider_ID'] = provider_id
+    return parsed_reputation
+
+
+def parse_raw_result(raw_result: Dict, file_hash: str, reliability: str, hash_type_key: str):
```

Comment:
The name isn't good. 
This functions is used solely by `files_reputations`, while there are other commands that have "raw_results".

Reply:
Should I name it `parse_file_reputation_raw_result` or `parse_file_reputation`?